client/blocks/editor-checkout-modal/index.tsx

```diff
@@ -82,29 +85,31 @@ const EditorCheckoutModal = ( props: Props ) => {
 		: cartData.products.map( ( product ) => product.product_slug );
 	const commaSeparatedProductSlugs = productSlugs?.join( ',' ) || null;
```

Comment:
```suggestion
	const commaSeparatedProductSlugs = productSlugs?.join( ',' );
```

I know it's not in the scope of this PR, but I noticed the TypeScript error while reviewing changes. The suggested change fixes the TypeScript error shown later in the same file when this variable is passed as the value of the `productAliasFromUrl` prop: `Type 'string | null' is not assignable to type 'string | undefined'`

cc @tjcafferkey to confirm this change won't break the composite checkout

Reply:
I actually opened #47798 to track this separately — going to mark this conversation as resolved. Sorry for the PR review noise